Agile_process.md

```diff
@@ -0,0 +1,287 @@
+be a UI campaign.
+
+## Process for proposing new sprints or campaigns
+
+To propose a new sprint, the process to follow is:
+
+1. Open a new issue on JIRA, and assign to it the issue type "Proposed Sprint" (we don't have a separate type for sprints and campaigns):
+
+![Proposed sprint screenshot](images/jira-proposed-sprint.png)
+
+2. Provide in the *Summary* field a one sentece description of the sprint.
+
+2. Fill out the *Description* field with a body of text according to the template in the section immediately below.  It should be relatively concise, but try to answer each of these questions clearly, so that the group leads and KBase executive committee can use this information to decide whether to accept your proposal and put it into the sprint pipeline.
+
+2. Assign it to Shane Canon, the production lead.
+
+2. List any components you may think it impacts, if you know in advance. 
+
+2. At this time, don't bother adding Epic/Theme or Sprint labels yet, as those will be added if the sprint is actually accepted for execution later on.
+
+### Template
+
+**Why it’s important:** Explain why KBase should do this.  How will it
+help make KBase unique, bring in users, improve the system, etc.  Why is
+it more important for KBase to do this compared to other priorities.
+
+**What needs to be done?** Provide a brief paragraph describing what has
+to be done including new data types, new UI pieces, example, etc.  Be
+complete but avoid going into implementation details.  This is more
+about understanding the scope of the activity versus exactly how it will
+be done.
+
+**Estimated Number of Sprints:** X sprints
+
+**Key Staff:** List important staff needed for the Sprint and what
+specific skills and knowledge they provide.
+
+**Dependencies:**  List any other campaigns, sprints, or activities
+that impact your proposed sprint or campaign.
+
+### Evaluation of sprint proposals
+
+Once a sprint/campaign proposal is in, there may be discussion and questions about it directly on the JIRA ticket.  The members of the KBase Executive Committee, who ultimately will make the decisions on which sprints to put into the implementation pipeline, will be added as watchers to the ticket so they have a chance to comment.
+
+Then, during the next maintenance week, the EC will include in its evaluation and re-prioritization decisions (that normally consider the outcome of the just-finished sprint cycle) any new proposals that have arrived.  This will allow the EC to weigh these proposals in the context of the outcomes and feedback of the recently concluded sprint cycle.
+
+The EC can decide to assign the proposals either to immediate execution in the next cycle, to be put in the queue for a later cycle, or it can decline the proposal.
+
+If a proposal is accepted, it will be given a sprint label and an epic/theme in JIRA so it can be sorted accordingly, and it will be re-assigned to its ultimate product owner.
```

Comment:
"Every accepted proposal results in an issue of type Sprint (we really need this type, imho, not just some random labels)."

In JIRA, we decided to use the built-in JIRA type "Epic", which is the JIRA/Agile term for campaigns. I could have created a new issue type "campaign" (and/or "sprint") but I used JIRA's built-in type, Epic, so that when we start using the JIRA Agile module, it works as intended, which it won't do if we make up our own issue types.
This was all discussed in email in late July.